### DISCUSSION TITLE
Prevent page reloading

### DISCUSSION THREAD

**Discussion by simonferndriger** [alpinejs/alpine#4429] [1. Help] [answered]
Hi

When you visit https://www.swisswebcams.ch/webcams/forch in a small browser (or click the `<-->` extension arrow to view the thumbnails in 1 column) and click on a webcam, it reloads the whole page completely if you use the back button from the browser (do it for a second time to avoid the ad overlay). Has this to do with the AplineJS framework? Can this reload be prevented?

In contrast, if you click on the back button of the web app itself (top left corner), the previous page appears instantly (due to AlpineJS routing).

Best
Simon

**Comment by ekwoka** [comment, marked as answer]
This isn't related to Alpine.

You would need to handle the the history `popstate` events.
